Dockerfile

```diff
@@ -44,11 +88,20 @@ RUN mkdir -p $ANDROID_HOME; \
 # Install Node.JS
 RUN cd /home/jenkins && \
     wget --output-document=nodejs.tgz --quiet \
-      'https://nodejs.org/dist/v8.9.3/node-v8.9.3-linux-x64.tar.xz' && \
+      'https://repository.cliqz.com/dist/android/nodejs/node-v8.9.3-linux-x64.tar.xz' && \
```

Comment:
why not to use nvm as you are using rvm already?

Reply:
True. Now using 'nvm'.